app/src/components/constants.js

```diff
@@ -46,13 +46,14 @@ module.exports = Object.freeze({
    */
   EMAILREGEX: '^[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\\.[a-zA-Z0-9-]{2,})+$',
 
+  /** Maximum number of parts supported by lib-storage upload */
+  MAXPARTCOUNT: 10000,
+
   /** Maximum Content Length supported by S3 CopyObjectCommand */
```

Comment:
i guess re-using this const is ok. I wondered if it could be named differently to make more sense when it's used in the different places. but i cant think of a better name.

but i would edit this comment to say:
`/** Maximum Content Length supported by S3 CopyObjectCommand and putObjectCommand*/`